README.md

```diff
@@ -837,9 +837,8 @@ func eatDoughnut(at index: Int) {
 
 ```swift
 // PREFERRED
-guard let monkeyIsland = monkeyIsland else {
-    return
-}
+guard let monkeyIsland = monkeyIsland else { return }
```

Comment:
Would this also apply if you were returning a value other then void?

Reply:
In this current proposal yes. See line 932 with a `// PREFERRED` example:
`guard let thingOne = thingOne else { return nil }`

Reply:
Ya, as we talked before, your comment got me thinking. In the sense that `return nil` is very different than `return foo?.value != nil ? true : false`.

Updated the proposal for only the first case (reduced return values)

Change on [52319ab](https://github.com/ecobee/swift-style-guide/pull/3/commits/52319ab2b7af77e6ce19041343285762652a735e).